### PR TITLE
Add tencent provider create-template

### PR DESCRIPTION
**Pull request by anycodes**
## What did you implement

Add Tencent provider create-template

## How can we verify it

* select a template, for example: tencent-python3
* run `cd tencent-python3` and create credentials `vim ~/credentials`, input your secretId, secretKey, appid:
```
[default]
tencent_appid = appid
tencent_secret_id = secretid
tencent_secret_key = secretkey
```
* deploy: `sls deploy`

### CODE REVIEW

File: lib/plugins/create/templates/tencent-nodejs8/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "version": "1.0.0",
+  "description": "Tencent Serverless Cloud Function example using Nodejs",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "cloud.tencent.com",
+  "license": "MIT",
+  "devDependencies": {
+    "serverless-tencent-scf": "*"
```
Comment (medikoo) [inline lib/plugins/create/templates/tencent-nodejs8/package.json]: It'll be good to pin it to current version so `^0.1.7`. That will prevent any eventual breaking changes to a plugin to affect the template

Comment (anycodes) [reply lib/plugins/create/templates/tencent-nodejs8/package.json]: Thank you.
I have added the plugin version in package.json just now.

File: lib/plugins/create/templates/tencent-nodejs6/gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# package directories
```
[the remark this answers is not in the document]
Comment (anycodes) [reply lib/plugins/create/templates/tencent-nodejs6/gitignore]: Thank you.  I have updated the template.

**Comment by medikoo** [comment]
@anycodes just noticed, there are some style issues. Can you run `npm run prettify-updated` and commit changes?

**Comment by anycodes** [comment]
DFOUNDERLIU-MB0:serverless dfounderliu$ npm run prettify-updated

> serverless@1.55.1 prettify-updated /Users/dfounderliu/Desktop/temp/serverless
> pipe-git-updated --ext=css --ext=html --ext=js --ext=json --ext=md --ext=yaml --ext=yml -- prettier --write

lib/plugins/create/create.js 135ms
lib/plugins/create/create.test.js 362ms
lib/plugins/create/templates/tencent-go/package.json 20ms
lib/plugins/create/templates/tencent-go/serverless.yml 58ms
lib/plugins/create/templates/tencent-nodejs/index.js 10ms
lib/plugins/create/templates/tencent-nodejs/package.json 6ms
lib/plugins/create/templates/tencent-nodejs/serverless.yml 16ms
lib/plugins/create/templates/tencent-php/package.json 8ms
lib/plugins/create/templates/tencent-php/serverless.yml 13ms
lib/plugins/create/templates/tencent-python/package.json 15ms
lib/plugins/create/templates/tencent-python/serverless.yml 12ms

**Pull request merged**